### ISSUE TITLE
20.5.3 Rolling joins, Figure 20.16

### ISSUE THREAD

**Issue by mitsuoxv** [hadley/r4ds#1470]
Figure 20.16 is inconsistent about which one is "closest".

I would like to suggest the following corrections:
1. Change the closest argument from key <= key to key >= key to make it compatible with figure caption.
2. Move a gray point from (1, 2) to (2, 1) in (x, y).

**Comment by mine-cetinkaya-rundel** [comment]
@hadley This also requires touching a diagram.

**Comment by florisvdh** [comment]
> Figure 20.16 is inconsistent about which one is "closest".
> 
> I would like to suggest the following corrections:
> 
> 1. Change the closest argument from key <= key to key >= key to make it compatible with figure caption.
> 2. Move a gray point from (1, 2) to (2, 1) in (x, y).

I agree (also noticed the error). Just note that the figure has now become fig. 19.16. Adding a screenshot to illustrate the problem. The proposed solution would indeed work; notice that it would also match the caption:

![afbeelding](https://github.com/hadley/r4ds/assets/19164640/9963606a-a705-4b86-8a66-3956246c516f)
